numpy/random/_bounded_integers.pyx.in

```diff
@@ -4,12 +4,54 @@
 import numpy as np
 cimport numpy as np
 
-from .distributions cimport *
-
 __all__ = []
 
 np.import_array()
 
+cdef extern from "include/distributions.h":
```

Comment:
Should these be in `_bounded_integers.pxd.in`?

Reply:
I don't think so, then we would have to ship `distributions.h`. But I think we should revisit this in terms of documented use-cases in #14604 after this goes in.